frontend/src/components/session/login-form.js

```diff
@@ -85,6 +90,9 @@ class LoginForm extends Component {
   render () {
     const { classes } = this.props
     const { type, action } = this.state
+    const passwordError = (this.state.password !== this.state.confirmPassword) ? (
```

Comment:
@nguyenquangv97 please use the Material-UI component for validation:
https://material-ui.com/components/text-fields/